app/models/company.rb

```diff
@@ -33,6 +33,9 @@ class Company < ApplicationRecord
 
   before_save { email.downcase! }
 
+  has_one :account, dependent: :nullify
+  has_one :address, as: :addressable, dependent: :nullify
```

Comment:
`dependent: :destroy`  this is a specific per company associations we dont need them without company

Reply:
done

Reply:
for address as well in all models

Reply:
done